src/LightSpeed/userFunction.js

```diff
@@ -38,6 +38,12 @@ const COMMON_SCOPE = {
         return min;
     },
 
+    pickOne(...choices) {
+        if (choices.length === 0)
+            throw new Error('You need at least one thing to pick from');
+        return choices[0];
```

Comment:
This is always picking the first element. Shouldn't it pick a random element?

Reply:
It should in Lightspeed, but not in the toolbox! It should always be deterministic. See the comment above this part, on `random()`. Otherwise you might get an error, and then have it be "fixed" by adding a space somewhere :stuck_out_tongue:

(it would be better to actually check _all_ the choices each time, but that's quite out of scope)

Reply:
Ah of course! I see now. /me is new to this part of the toolbox code.